src/panels/lovelace/editor/card-editor/hui-card-element-editor.ts

```diff
@@ -1,11 +1,13 @@
-import { customElement } from "lit/decorators";
+import { customElement, property } from "lit/decorators";
 import type { LovelaceCardConfig } from "../../../../data/lovelace";
 import { getCardElementClass } from "../../create-element/create-card-element";
 import type { LovelaceCardEditor } from "../../types";
 import { HuiElementEditor } from "../hui-element-editor";
 
 @customElement("hui-card-element-editor")
 export class HuiCardElementEditor extends HuiElementEditor<LovelaceCardConfig> {
+  @property() public clipboard?: LovelaceCardConfig;
```

Comment:
I would suggest to use `sessionStorage` here, like we also do in the automation editor, this also solves the issues of passing the clipboard around, as any element can then just get it from there, and if they subscribe are also updated.